eslint.config.ts

```diff
@@ -101,14 +104,13 @@ export default defineConfig([
   pluginVue.configs['flat/recommended'],
   eslintPluginPrettierRecommended,
   storybook.configs['flat/recommended'],
-  // @ts-expect-error Bad types in the plugin
   importX.flatConfigs.recommended,
-  // @ts-expect-error Bad types in the plugin
   importX.flatConfigs.typescript,
+  // Turn off ESLint rules that are already handled by oxlint
+  oxlint.configs['flat/recommended'],
```

Comment:
@snomiao hello from `eslint-plugin-oxlint` contributor
Want to share some info:

Only one `oxlint.` call is needed. `flat/recommended` will disable more rules than enabled in your config. 

2 Solutions:
- Enable more oxlint plugins (import, vue, unicorn, typescript) and use `flat/recomended` instead of `oxlint.buildFromOxlintConfigFile`
- Remove `flat/recommended` and just use `oxlint.buildFromOxlintConfigFile`

The return value is an array, use the spread syntax (see the docs) :)